src/rss/rss-extensions/sesamy.ts

```diff
@@ -20,10 +20,12 @@ export const sesamyProductSchema = z.object({
   description: z.string().optional(),
   image: z.string().optional(),
   link: z.string().optional(),
-  type: z.string(),
-  period: z.enum(['day', 'week', 'month', 'year']).optional(),
+  // @deprecated
+  type: z.string().optional(),
+  period: z.enum(['DAY', 'WEEK', 'MONTH', 'YEAR']).optional(),
   price: z.number(),
-  purchase_type: z.enum(['own', 'lease', 'recurring']).optional(),
+  product_type: z.enum(['ARTICLE', 'EPISODE', 'PODCAST', 'PASS', 'BUNDLE']).optional(),
+  purchase_type: z.enum(['OWN', 'LEASE', 'RECURRING']).optional(),
```

Comment:
Can we make `product_type` & `purchase_type` non-optional here?

I will be relying on `purchase_type` atleastto always be something.

Reply:
Not quite yet as it would be a breaking change..

Reply:
But these ones are new for `parser` right? So we can enforce them.

Or do these have to be optional because they are in the backend?

We should really start enforcing stuff like this, otherwise we have to provide graceful fallbacks / error handlings throughout the entire chain.

Ping @majkiee @nachojimenez

Reply:
It's used in a bunch of places so we can't just start making breaking changes. Once the proxy and the parser is updated (and we make sure it doesn't break consume) we can change so it's enforced